tests/conftest.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Neal Chambers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List, Union
+
+import pytest
+
+
+@pytest.fixture(scope="session")
```

Comment:
Can you add a comment describing what this file does?

Reply:
Unresolving: the comment is not there yet.

Reply:
This is resolved locally, just waiting for your response to the python question before I commit.